src/redux/userLists.ts

```diff
@@ -114,44 +198,46 @@ export const userListsUpdateList = (assetAddress, listId, add = true) => (
     });
 
     // add or remove
-    const updatedListTokens = add
-      ? uniq(concat(allNewLists[listIndex].tokens, assetAddress))
-      : isArray(assetAddress)
-      ? without(allNewLists[listIndex].tokens, ...assetAddress)
-      : without(allNewLists[listIndex].tokens, assetAddress);
-    if (add) {
-      dispatch(emitAssetRequest(assetAddress));
-    }
+    if (listIndex) {
```

Comment:
`listIndex` could be `0`, right? Should this be `if (listIndex !== null)`?

Reply:
ah yes very good point - nice catch!

Reply:
Yes definitely, I updated that as well as the one below, thanks!